config/prod/namespaced/s3-to-glue-lambda-role.yaml

```diff
@@ -0,0 +1,7 @@
+template:
```

Comment:
The only issue is that the lambda role here needs the sqs queue as a dependency and the sqs-queue is namespaced. This is because inside the lambda role's cloudformation, it only gives polling permissions to the namespaced sqs queue. So i think this is why we would need to keep it namespaced

Reply:
I see what you mean. Let's keep it namespaced, then.